osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/GetPowerQualityProfileResponseDataDto.java

```diff
@@ -0,0 +1,39 @@
+package org.opensmartgridplatform.dto.valueobjects.smartmetering;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GetPowerQualityProfileResponseDataDto extends ActionResponseDto {
+
+    private static final long serialVersionUID = -156966569210717654L;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GetPowerQualityProfileResponseDataDto.class);
+
+    public List<PowerQualityProfileDataDto> getPowerQualityProfileResponseDataDtos() {
+        return this.responses;
+    }
+
+    private final List<PowerQualityProfileDataDto> responses;
+
+    public GetPowerQualityProfileResponseDataDto() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1213)